pkg/asset/internal/templates.go

```diff
@@ -216,7 +218,7 @@ spec:
       hostNetwork: true
       containers:
       - name: checkpoint-installer
-        image: quay.io/coreos/pod-checkpointer:5b585a2d731173713fa6871c436f6c53fa17f754
+        image: quay.io/coreos/pod-checkpointer:e054c5b5a95993baa15ed94e2e2083cc826d4d49
```

Comment:
I cut a new release of checkpointer which also contains support for configMaps (https://github.com/kubernetes-incubator/bootkube/pull/320). We could bump to that release here if you're not opposed (we can also wait and bump separately).